example/processors/csv_checker.py

```diff
@@ -29,19 +30,19 @@ def check_character_blocks(csv):
 
     if None in block_set:
         block_set.remove(None)
-        tabular_add_issue(
+        report.TablularReport.add_issue(
```

Comment:
Not too sure I understand this bit - is there a report object being created, as in boundary_checker_impr.py, for example, or is it just being statically called? Also, think there's a typo here (Tablular) - if this does run correctly, there must be a typo also in report.py?

Reply:
actually, from the get_workflow call, it looks as if you maybe want to add another argument to this function called `r` or `rprt`, and pass self._report to `check_character_blocks`, as well as passing `csv` - this is like the change you made in line 119 (i.e. adding self._report to the list of arguments in line 118, and changing the arguments to `(csv, rprt)` in line 23, say)

Reply:
if that's confusing, happy to chat through!